ddtrace/internal/datadog/profiling/include/exporter.hpp

```diff
@@ -130,7 +131,7 @@ class Profile {
   // strategy to dedup + store strings
   using StringTable = std::unordered_set<std::string_view>;
```

Comment:
why not use directly the unordered_set as storage ? References to values are guaranteed to be stable.

Reply:
I wrote a comment, but it looks like GitHub ate it up.  Writing succinctly.

The version of gcc we use in CI doesn't support heterogeneous access for unordered_set.  That means I'd need to convert the incoming `string_view` into a `string` in order to do any find-type comparison.  I was considering this to be a deal-breaker, since I expect the overall string duplication rate to be high.

Updating gcc is easy, but it isn't an option because some of the other extensions fail their builds on newer-GCC and I don't want to investigate all of that.

Arguably, I could use a different backing store.  :shrug: